libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -6,7 +6,12 @@
 --
 
 You can use the {stack} {monitor-features} to gain insight into the health of
+ifndef::apm-server[]
 {beatname_uc} agents running in your environment. 
```

Comment:
```suggestion
{beatname_uc} instances running in your environment. 
```

Reply:
Changed this to be consistent with a naming discussion we had in a different PR.